README.md

```diff
@@ -57,16 +57,43 @@ pywal
 pacman-contrib
 ```
 
-Arch (pacman):
+### Arch
+
+pacman:
 ```bash
 sudo pacman -S pipewire libgtop bluez bluez-utils btop networkmanager dart-sass wl-clipboard brightnessctl swww python gnome-bluetooth-3.0 pacman-contrib
 ```
 
-Arch (AUR):
+AUR:
 ```bash
 yay -S grimblast-git gpu-screen-recorder hyprpicker matugen-bin python-gpustat aylurs-gtk-shell-git
 ```
 
+### Fedora
+
+DNF - with solopasha's [hyprland COPR](https://copr.fedorainfracloud.org/coprs/solopasha/hyprland/)
+```bash
+dnf copr enable solopasha/hyprland
```

Comment:
Add `sudo` to the dnf and flatpak install instructions to be consistent with arch install instructions (yay doesn't need sudo).